### PR TITLE
MGMT-7307: Improve the disconnected media message to explain what is …

### PR DESCRIPTION
**Pull request by michaellevy101**
…expected from the user

# Assisted Pull Request

## List all the issues related to this PR

- [ ] New Feature
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [x] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Assignees

/cc @
/cc @

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] Reviewers have been listed
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- [ ] Are the title and description (in both PR and commit) meaningful and clear?
- [ ] Is there a bug required (and linked) for this change?
- [ ] Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: internal/bminventory/inventory.go [hunk 1]
```diff
@@ -76,7 +76,7 @@ import (
 const DefaultUser = "kubeadmin"
 
 const WindowBetweenRequestsInSeconds = 10 * time.Second
-const mediaDisconnectionMessage = "Cannot read from the media (ISO) - media was likely disconnected"
+const mediaDisconnectionMessage = "Unable to read from the discovery media.  It was either disconnected, or poor network conditions prevented it from being read.  Try using the minimal ISO option, and be sure to keep the media connected until the installation completes"
```
Comment (empovit) [inline internal/bminventory/inventory.go:79]: There are double-whitespaces in the message. Is this by design?

Comment (michaellevy101) [reply internal/bminventory/inventory.go:79]: @avishayt

Comment (romfreiman) [reply internal/bminventory/inventory.go:79]: Just remove the double spaces :)

**Pull request merged**